sample.xml

```diff
@@ -5,46 +5,174 @@
+geo can be a POINT, a CIRCLE, or a POLYGON, and the function will return a
+geometry of the same type.  In the current implementation, from_sys and
+to_sys must be literal strings (i.e., they cannot be computed through
+expressions or be taken from database columns).
+
+All transforms are just simple rotations, which is only a rough 
+approximation to the actual relationships between reference systems
+(in particular between FK4 and ICRS-based ones).  Note that, in particular,
+the epoch is not changed (i.e., no proper motions are applied).
+
+We currently support the following reference frames: ICRS, FK5 (which
+is treated as ICRS), FK4 (for B1950. without epoch-dependent corrections),
+GALACTIC.  Reference frame names are case-sensitive.</description>
+        </feature>
+        <feature>
+          <form>gavo_vocmatch(vocname TEXT, term TEXT, matchagainst TEXT) -&gt; INTEGER</form>
+          <description>returns 1 if matchagainst is term or narrower in the IVOA vocabulary
+vocname, 0 otherwise.
+
+This is intended for semantic querying.  For instance, 
+gavo_vocmatch('datalink/core', 'calibration', semantics) would be 1
+if semantics is any of calibration, bias, dark, or flat.
+
+For RDF-flavoured vocabularies (strict trees), term will expand to the
+entire branch rooted in term.  For SKOS-flavoured vocabularies (where
+narrower is not transitive), only directly narrower terms will be included.
+
+Both the term and the vocabulary name must be string literals (i.e.,
+constants).  matchagainst can be any string-valued expression.</description>
+        </feature>
+        <feature>
+          <form>ivo_healpix_center(hpxOrder INTEGER, hpxIndex BIGINT) -&gt; POINT</form>
+          <description>returns a POINT corresponding to the center of the healpix with
+the given index at the given order.</description>
+        </feature>
+        <feature>
+          <form>ivo_healpix_index(order INTEGER, ra DOUBLE PRECISION, dec DOUBLE PRECISION) -&gt; BIGINT</form>
+          <description>Returns the index of the (nest) healpix with order containing the 
+spherical point (ra, dec).
+
+An alternative, 2-argument form 
+
+ivo_healpix_index(order INTEGER, p POINT) -&gt; BIGINT
+
+is also available.</description>
+        </feature>
+        <feature>
+          <form>ivo_interval_has(val NUMERIC, iv INTERVAL) -&gt; INTEGER</form>
+          <description>The function returns 1 if the interval iv contains val, 0 otherwise.
+The lower limit is always included in iv, behaviour on the upper
+limit is column-specific.</description>
+        </feature>
+        <feature>
+          <form>ivo_interval_overlaps(l1 NUMERIC, h1 NUMERIC, l2 NUMERIC, h2 NUMERIC) -&gt; INTEGER</form>
+          <description>The function returns 1 if the interval [l1...h1] overlaps with
+the interval [l2...h2].  For the purposes of this function,
+the case l1=h2 or l2=h1 is treated as overlap.  The function
+returns 0 for non-overlapping intervals.</description>
```

Comment:
For the purposes of a sample document, I appreciate there being some custom functions here, but if we want to keep it a little shorter (and less daunting) we can (should?) delete a few.